libs/ml/include/ml/utilities/graph_builder.hpp

```diff
@@ -385,6 +386,13 @@ void BuildNodeAndInsertTrainables(NodeSaveableParams<T> const &nsp, std::string
     g->AddTrainable(node, name);
     break;
   }
+  case ops::ReduceMean<T>::OpCode():
```

Comment:
nice, well remembered! 👍 

we should add a test that builds a graph with one of every type of op, serialises it, then deserialises it. That should help us avoid missing these in the future. I'll add a task on jira to this effect.